sensors/SpecCoSensor.py

```diff
@@ -22,3 +22,16 @@ def format_data(self) -> dict:
         }
 
         return reading
+    
+    def get_data(self) -> dict:
+        reading_dict = {
+            "uid": self.get_uid(),
+            "serial_number": self.get_serial_number(),
+            "co": ppb_to_ppm(self.get_measurement()),
```

Comment:
change "co" to "CO" to conform to condensed structural formula specification (introduced a few days ago)